lizafloyd/portfolio_site
# Remove eslint

**Description**
> Provide a description of what your changes do.

It looks like the `create-react-app` package already installs `eslint`, and requires a very specific version to work. What happened is that dependabot tried to install an incompatible version which was causing incorrect errors. To fix this I removed `eslint` from the package.json dependencies as it's already installed via the create-react-app script.

If a dependency update _breaks_ something the in the application, you can indicate to Dependabot that you don't want it to auto update by using the `=` syntax.

For example (Look at Enzyme in the following):

```json
  "devDependencies": {
    "enzyme": "=3.9.0",
    "enzyme-adapter-react-16": "*",
    "eslint-config-prettier": "^4.0.0",
    "eslint-plugin-jsx-a11y": "^6.2.1",
    "eslint-plugin-react": "^7.12.3",
    "gh-pages": "^2.0.1",
    "node-sass": "^4.11.0",
    "prettier": "^1.16.4",
    "react-test-renderer": "^16.8.2",
    "sass-lint": "^1.12.1"
  },
```

In this case `eslint` isn't actually needed here, so I just removed it. The build should pass now.


**Testing Instructions**
> Give step by step instructions on how to test your changes.

* Run `yarn format && yarn lint && yarn test` and make sure that everything still works correctly.
